scania-digital-design-system/sdds
# refactor(button): move styling from themes package to component

**Describe pull-request**  
- moved styling from themes package to component package
- replaced `$prefix` to `--sdds`

**Solving issue**  
Fixes: [AB#1681](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/1681)

**How to test**  
1. Go to storybook button story 
2. Check buttons if it looks same as current ones
